apps/src/lib/config.rs

```diff
@@ -233,6 +232,40 @@ impl Default for IntentGossiper {
     }
 }
 
+#[cfg(any(test, feature = "testing"))]
```

Comment:
If you could pls move this `impl` to the bottom of the file to have the non-testing code first